custom_components/smartthinq_sensors/wideq/ac.py

```diff
@@ -882,11 +903,23 @@ def duct_zones_type(self):
             return None
         return self.to_int_or_none(self._data.get(DUCT_ZONE_V1_TYPE))
 
+    @property
+    def mode_jet(self):
+        key = self._get_state_key(STATE_MODE_JET)
+        if self._data.get(key) == self.to_int_or_none(MODE_JET_COOL) or self._data.get(
+            key
+        ) == self.to_int_or_none(MODE_JET_HEAT):
+            value = True
+        else:
+            value = False
+        return self._update_feature(FEAT_MODE_JET, value, False)
```

Comment:
Here is better to check if `jet mode` is supported, so that device that not support it will not have the feature and the switch:

```
        if (value := self.to_int_or_none(self._data.get(key))) is None:
            return None
        status = value in (MODE_JET_COOL, MODE_JET_HEAT)
        return self._update_feature(FEAT_MODE_JET, status, False)
```

Reply:
Done!